javascript/javascript1/week1/Age-ify.js

```diff
@@ -0,0 +1,4 @@
+const yearOfBirth = 1985;
+let yearFuture = 2040;
```

Comment:
use const to declare the variable unless you want to re-assign the value of the variable.

Reply:
done . thank you.